matrix-meetings-bot/test/EventContentRenderer.test.ts

```diff
@@ -47,6 +48,44 @@ describe('EventContentRenderer', () => {
     }
     return widgetContent;
   };
+  test('should validate base32_room_id50 correctly', () => {
+    const params = {
+      room_id: 'room1',
+      base32_room_id: 'room1',
+      base32_room_id50: 'roomId50',
+      title: 'Demo Room',
+    };
+
+    // Ensure that only the base32_room_id50 property has a length of 50 characters
+    const valid = Object.keys(params).every((key) => {
+      if (key === 'base32_room_id50') {
+        return params[key].length <= 50;
+      }
+      return true;
+    });
+
+    expect(valid).toBe(true);
+  });
```

Comment:
I think we should test `EventContentParamsHelper.newInstance` with `base32_room_id50` in a separate spec, not here.

Reply:
i can't any meaningful place to put the tests. do you think i should create a new test file?

Reply:
i created a new spec file and reverted the tests with the validation in this file

Reply:
thanks for the change, okay